Misc/NEWS.d/next/Library/2023-02-04-16-35-46.gh-issue-101561.Xo6pIZ.rst

```diff
@@ -0,0 +1,5 @@
+Add a new decorator ``typing.override``.
+
+This decorator indicates to static type checkers that they should verify the method overrides an attribute or method of the same name in a base class. This is useful for catching bugs when base class methods are renamed but some children are not updated accordingly.
+
+See :pep:`698` for details. Patch by Steven Troxler.

```

Comment:
```suggestion
Add a new decorator :func:`typing.override`. See :pep:`698` for details. Patch by Steven Troxler.
```
This can be kept more concise. The What's New can have more of an explanation.